pkg/cli/updater/updater.go

```diff
@@ -153,6 +154,7 @@ func (u *updater) defaultOptions() error {
 		var err error
 		u.executablePath, err = exec.ResolveExecuable(os.Args[0])
 		if err != nil {
+			u.hookSkipUpdateIntoCLI()
```

Comment:
Question: is there a use case to pass `--skip-updates` flag when returning error?

Reply:
I'm not sure about this one on the line 157. But for sure there it makes sense for the one on line 167. And I did not want to make a difference, the skip update parameter simply should be always accepted. The error is logged but does not prevent execution anyway.